src/test/java/org/mule/extension/file/integration/FileWriteTestCase.java

```diff
@@ -206,4 +238,27 @@ private String doWriteOnExistingFile(FileWriteMode mode) throws Exception {
     doWrite(file.getAbsolutePath(), HELLO_WORLD, mode, false);
     return readPathAsString(file.getAbsolutePath());
   }
+
+  public static InputStream getContentStream() {
+    return (new InputStream() {
+
+      String text = "Hello World!";
+      char[] textArray = text.toCharArray();
+      int index = -1;
+
+      @Override
+      public int read() throws IOException {
+        try {
+          Thread.sleep(10);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2925))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-file-connector&pullRequest=97&issues=AWkMG7u6otIXnmsUszLl&open=AWkMG7u6otIXnmsUszLl)

[ISSUE_KEY]: AWkMG7u6otIXnmsUszLl

Reply:
It is needed

Reply:
It is needed